docs/simulations/mathematics/LissajousPattern.md

```diff
@@ -0,0 +1,34 @@
+# Lissajous Pattern
+---
+<div style="float:right">
+  ![LissajousPatternLight](https://raw.githubusercontent.com/cod-ed/simulate/master/assets/simulations/LissajousCurveLight.png){: style="height:200px";}
+</div>
+
+<p align="justify">
+A Lissajous curve in mathematics, also known as Lissajous Figure or Bowditch curve is the
+graphical representation of the following system of parametric equations.</p>
+
+<center>
+$x = A sin(at + δ)$  
+$y = B sin(bt)$
+</center>
+  
+These curves/patterns are highly sensitive to the values of $‘a’$ and $‘b’$, particularly to the ratio
+$a:b$.  
+For a ratio of $1$, and $δ = \frac{π}{2} radians$, the figure is a *circle*. Similarly, for $a = b$ and $δ = 0$, the figure is a *line*.
+Another simple Lissajous figure is the *parabola* $(\frac{b}{a} = 2, δ = \frac{π}{4})$.
+
+Lissajous curves can also be generated using an oscilloscope. An octopus circuit can be used
+to demonstrate the waveform images on an oscilloscope. Two phase-shifted sinusoid inputs are
+applied to the oscilloscope in X-Y mode and the phase relationship between the signals is
+presented as a Lissajous figure. For convenience, the values of $‘A’$ and $‘B’$ are equal and
+constant in the app. A beautiful animation of how the pattern is drawn is also available.
```

Comment:
I don't think this is actually related to the simulation so we can get rid of it, maybe add only the point about A and B, however, I feel it isn't needed too.